src/screens/OrganizationFundCampaign/OrganizationFundCampagins.tsx

```
@@ -186,39 +195,43 @@ const orgFundCampaign = (): JSX.Element => {
-      },
+      renderCell: (params: GridCellParams) => (
+        <div
+          className={styles.hyperlinkText}
+          data-testid="campaignName"
+          onClick={() => handleClick(params.row.id as string)}
+          onKeyDown={(e) => {
+            if (e.key === 'Enter' || e.key === ' ') {
+              handleClick(params.row.id as string);
+            }
+          }}
+          role="button"
```

Biome (1.9.4)

[error] The elements with the following roles can be changed to the following elements:
<button>
<input type="button">


For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)